tripal/includes/TripalEntityUIController.inc

```diff
@@ -876,11 +901,56 @@ function theme_tripal_add_list($variables) {
   return $output;
 }
 
+/**
+ * Form callback: confirmation form for unpublishing a tripal_entity.
+ *
+ * @param $tripal_entity
+ *   The tripal_entity to delete
+ *
+ * @see confirm_form()
+ */
+function tripal_entity_unpublish_form($form, &$form_state, $entity) {
+  $form_state['entity'] = $entity;
+  $form['#submit'][] = 'tripal_entity_unpublish_form_submit';
+
+  $form = confirm_form($form,
+    t('Unpublish the record "%title"?',
```

Comment:
Maybe a sentence to match the delete content message?

```suggestion
    t('Unpublish the record "%title"?  This will delete the Drupal record, but not the underlying (eg Chado) record.',
```